src/OSPSuite.Core/Domain/Services/PKCalculationOptions.cs

```diff
@@ -14,45 +18,9 @@ public class PKCalculationOptions
       /// </summary>
       public double? InfusionTime { get; set; }
 
-      /// <summary>
-      ///    Time value representing the beginning of first dosing interval in time array
-      /// </summary>
-      public float? FirstDosingStartValue { get; set; }
+      public IReadOnlyList<DosingInterval> DosingIntervals => _dosingIntervals;
 
-      /// <summary>
```

Comment:
We had in essense 3 fix intervals. I have generalized the concept to n intervals